docs/one-wire-devices.md

```diff
@@ -0,0 +1,169 @@
+
+## Connect a One-Wire Device to the Omega2
+
+This section covers a three-stage process for establishing the physical wiring connection of your One-Wire device to GPIO19, where the latter will serve the role of the One-Wire master.
+
+### Stage 1: Identify the Connectors
+
+One-Wire devices have three available connectors/wires as illustrated in the table below: 
+
+| Wire            | Connection                                                      |
+|-----------------|-----------------------------------------------------------------|
+| VCC             | Connect this wire to the 3.3V power source on the Omega2 board. |  
+| GND             | Connect this wire to the ground pin on the Omega2 board.        |  
+| DQ              | Connect this wire to the GPIO19 pin on the Omega2 board.        | 
+
+![one wire device circuit diagram](../static/img/one-wire-device-circuit-diagram-1.png)
+
+:::note
+
+Please refer to your specific One-Wire device's datasheet to identify the pins and determine the recommended voltage. 
+
+:::
+
+:::tip
+
+Building these connections is relatively easy if you have an Expansion, Power, or Arduino Dock since they all expose the Omega’s GPIOs. 
+
+:::
+
+:::warning
+
+Some One-Wire devices will require a pull-up resistor on the Data line (DQ). For example, the popular DS18B20 temperature sensor requires a 4.7 kΩ pull-up resistor on the DQ to operate properly.
+
+:::
+
+
+### Stage 2: Connect Power & Ground
+
+Connect the power and ground wires belonging to your One-Wire device to the corresponding power and ground pins on the Omega2 board. 
+
+### Stage 3: Connect the DQ
+
+Connect the DQ from your One-Wire device to the GPIO19 pin on the Omega2 board. 
+
+## Enable One-Wire Device Support
+
+
+Once the physical connections are made, install dts overlay package to register a one-wire master in Linux associated with GPIO19 to communicate with the one-wire slave device.  
+
+**Step 1:**  Install the `onion-dt-overlay-w1-gpio` package using opkg
+
+```
+opkg install onion-dt-overlay-w1-gpio
+```
+**Step 2:** After the package is successfully installed, GPIO19 will act as a One-Wire Master. 
+
+```
+/sys/devices/w1_bus_master1
+```
+<h2 id="find">Find Connected One-Wire Devices</h2>
+
+**Step 1:** Navigate to the relevant directory where One-Wire devices are listed. This is under `/sys/devices/w1_bus_master1`
+
+
+```
+cd /sys/devices/w1_bus_master1
+ls
+```
+
+**Step 2:** Perform a check to find if slave devices are connected or not. 
+
+```
+cat /sys/devices/w1_bus_master1/w1_master_slave_count
+```
+
+**Step 3:** The command in step 2 will return the count of the number of slave devices connected. 
+
+| Result            | Interpretation                            |
+|-------------------|-------------------------------------------|
+| 1                 | A slave device is connected               |
+| 0                 | A slave device is connected               |
```

Comment:
We say in the description this will return the number of connected one-wire devices but then only list 0 and 1 as viable results.
Please add a row to the table for 1+ or other - whatever makes sense to you